src/libraries/System.ComponentModel.Composition/src/System/ComponentModel/Composition/Hosting/AtomicComposition.cs

```diff
@@ -14,7 +14,7 @@ namespace System.ComponentModel.Composition.Hosting
     /// queryable state along with a sequence of actions necessary to complete the state when
     /// the atomicComposition is no longer in danger of being rolled back.  State is completed or
     /// rolled back when the atomicComposition is disposed, depending on the state of the
-    /// CompleteOnDipose property which defaults to false.  The using(...) pattern in C# is a
+    /// CompleteOnDispose property which defaults to false.  The using(...) pattern in C# is a
```

Comment:
There is not such property (even in the initial MEF1 port https://github.com/dotnet/runtime/commit/a6f350fabbcce17c84dc396dd744d9651ad1ca25).